terraform-build-user/main.tf

```
@@ -9,6 +9,6 @@ module "iam_user" {
     aws.images-staging-ssm    = aws.images-staging-ssm
   }
 
-  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*"]
+  ssm_parameters = ["/cyhy/dev/users", "/ssh/public_keys/*", "/windows/commando/administrator/password"]
```

This should be the only value this build user needs access to successfully building an image.

```suggestion
  ssm_parameters = ["/windows/commando/administrator/password"]
```

---

@mcdonnnj now that this repo won't have commando, what about renaming the remaining ssm parameter to `/windows/server/administrator/password`? If so, I believe that would need to be changed on your end and i'll have this code updated.

---

@mcdonnnj resolved with [ec8bb5a](https://github.com/cisagov/windows-server-packer/pull/10/commits/ec8bb5a2ab09498a1a78db8d27262633013fc270)